.gitignore

```diff
@@ -55,3 +55,4 @@ docs/_build/
 
 *.so
 /venv
+.eggs
```

Comment:
Pythons reproduce through eggs ;) http://stackoverflow.com/questions/2051192/what-is-a-python-egg The `.eggs` folder happens to be some sort of cache, I think.